src/granite/transactions.cr

```diff
@@ -125,9 +125,15 @@ module Granite::Transactions
 
     private def __update
       set_timestamps mode: :update
-      fields = self.class.content_fields
+      fields = self.class.content_fields.dup
       params = content_values + [@{{primary_name}}]
 
+      # Do not update created_at on update
+      if created_at_index = fields.index("created_at")
+        fields.delete_at created_at_index
+        params.delete_at created_at_index
```

Comment:
There is opportunity for this code to crash when fields and params fall out of sync. The resulting bug will be harder to find because a seemingly random index in params will have been deleted.

Could this conditional be replaced with two `.delete("created_at")` calls instead?

Reply:
I'm open to ideas.  But currently its a bit tricky since `fields` is just an array of column names, and `params` is just an array of values.  It would be much easier if it was like a hash of `column_name => value`.  

That might not be a bad idea, but would deff increase the scope of the PR.

Reply:
Oh geeze, I didn't realize that. It's not worth unwinding here.